src/network/websockets/index.ts

```diff
@@ -100,7 +107,7 @@ export function makeWsProtocolAdapter(
             id,
             pushId,
             lastSeenRevision,
-            value: serialization.encode(value)
+            value: serialization.encode(value, { id, kind: String(kind) })
```

Comment:
@denis-sokolov is this correct? I haven't had time to dive too much into some of the type definitions but the type checker mentioned kind could be a few other type other than a string. Is converting to a string here ok?

Reply:
`kind` variable comes from the definition of our type, so its `keyof Domain`.

https://github.com/citrusbyte/oncilla/blob/d649bd2341a047ef71aaf6415c78048c9e68cff1/src/network/types.ts#L18-L19

Technically we could type it more strictly and precisely, but we’re doing very dynamic things and as long as our API is clean type-wise, it’s not a big deal. You’ll find a few `as any` in this file, too.

Let’s leave it like this for now and maybe later we’ll tighten up the types.